rom/laghos.cpp

```diff
@@ -1727,14 +1817,30 @@ int main(int argc, char *argv[])
                     {
                         romOper[romOptions.window]->ApplyHyperreduction(romS);
                     }
+
+                    if (problem == 7 && romOptions.indicatorType == penetrationDistance)
+                    {
+                        std::string pd_weight_outPath = testing_parameter_outputPath + "/pd_weight" + to_string(romOptions.window);
```

Comment:
Lines 1823-1833 are the same as lines 1271-1281 except for the window number. This could be made into a function, to avoid code duplication.

Reply:
Changes have been made accordingly.